api/src/main/resources/liquibase.xml

```diff
@@ -362,6 +365,9 @@
 		<modifySql dbms="mssql">
 			<replace replace="name-of-uuid-function" with="NEWSEQUENTIALID()"/>
 		</modifySql>
+		<modifySql dbms="postgresql">
+			<replace replace="name-of-uuid-function" with="UUID()"/>
```

Comment:
Don't we already have the UUID() function already taken care of in openmrs-core?

Reply:
Yes, that is why we are using UUID() here and not uuid_generate_v1(). In openmrs-core we have created the wrapper function similar to MySQL for PostgreSQL which is UUID() that wraps around uuid_generate_v1().